libs/barista-components/slider/src/slider.scss

```diff
@@ -67,6 +67,10 @@
 .dt-slider-fill {
   background-color: $turquoise-600;
   transform-origin: 0 0;
+
+  .dt-disabled & {
+    background-color: $gray-300;
+  }
```

Comment:
NIT: I'd rather create a readable selector and have duplicate code. This way it's hard to understand if the selector is 
`.dt-slider-fill .dt-disabled {` or `.dt-disabled .dt-slider-fill` nesting suggests the first and the ampersand the latter

Reply:
I was fond of this sassy sass code, but I will update it.